pkg/apis/core/validation/shoot.go

```diff
@@ -1877,8 +1877,11 @@ func ValidateCoreDNSRewritingCommonSuffixes(commonSuffixes []string, fldPath *fi
 
 	suffixes := map[string]struct{}{}
 	for i, s := range commonSuffixes {
-		if strings.Count(s, ".") < 2 {
-			allErrs = append(allErrs, field.Invalid(fldPath.Child("commonSuffixes").Index(i), s, "not enough dots ('.'), at least two dots required"))
+		if strings.Count(s, ".") < 1 {
+			allErrs = append(allErrs, field.Invalid(fldPath.Child("commonSuffixes").Index(i), s, "not enough dots ('.'), at least one dot required"))
+		}
+		if strings.HasPrefix(s, ".") {
+			allErrs = append(allErrs, field.Invalid(fldPath.Child("commonSuffixes").Index(i), s, "should not start with a dot ('.'), as it is automatically appended"))
```

Comment:
This is a breaking change, isn't it? What if `Shoot`s exist which have a leading `.`?

Reply:
Yes, I will change it. 
I will allow the leading dot and remove it later while building the regex.